pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraLocation.scala

```diff
@@ -69,23 +69,28 @@ trait SierraLocation extends SierraQueryOps {
         .subfieldsWithTag("f")
         .contents
         .headOption
-        .flatMap {
+        .map {
           case accessStatus(status) if status == "Open" =>
-            Some(AccessStatus.Open)
+            AccessStatus.Open
           case accessStatus(status) if status == "Open with advisory" =>
-            Some(AccessStatus.OpenWithAdvisory)
+            AccessStatus.OpenWithAdvisory
           case accessStatus(status) if status == "Restricted" =>
-            Some(AccessStatus.Restricted)
+            AccessStatus.Restricted
           case accessStatus(status)
               if status == "Restricted access (Data Protection Act)" =>
-            Some(AccessStatus.Restricted)
+            AccessStatus.Restricted
           case accessStatus(status) if status == "Cannot Be Produced" =>
-            Some(AccessStatus.Restricted)
+            AccessStatus.Restricted
           case accessStatus(status) if status == "Certain restrictions apply" =>
-            Some(AccessStatus.Restricted)
+            AccessStatus.Restricted
           case accessStatus(status) if status == "Closed" =>
-            Some(AccessStatus.Closed)
-          case _ => None
```

Comment:
Does removing this mean this is now a partial pattern match, and will error if something unexpected?

Reply:
yes, I modified the corresponding test. That was a bit of a workaround to get a usable index again, but I think we want to know if there are other mappings to add